### ISSUE TITLE
No video/window shows up on Hyprland

### ISSUE THREAD

**Issue by gptlang** [FDH2/UxPlay#311]
```
...
raop_rtp video: now = 1718295221.310789, ntp = 1718295221.412842, latency = -0.102052, ts = 30630.157392, 00 00 00 00 
video renderer resumed
raop_rtp video: now = 1718295221.344121, ntp = 1718295221.446174, latency = -0.102053, ts = 30630.190725, 00 00 00 00 
video renderer resumed
raop_rtp video: now = 1718295221.380321, ntp = 1718295221.479506, latency = -0.099184, ts = 30630.224057, 00 00 00 00 
video renderer resumed
raop_rtp video: now = 1718295221.451534, ntp = 1718295221.512838, latency = -0.061304, ts = 30630.257389, 00 00 00 00 
video renderer resumed
raop_rtp video: now = 1718295221.451607, ntp = 1718295221.546170, latency = -0.094563, ts = 30630.290721, 00 00 00 00 
video renderer resumed
...
```
Logs show that everything is fine

**Comment by gptlang** [comment]
I've tried `env -u WAYLAND_DISPLAY uxplay` as well so it can't be a wayland issue

**Comment by fduncanh** [comment]
seems related to #284
 give details of 
1. your distribution (e.g Ubuntu, Arch etc)
2.  gstreamer version   (probably 1.24.x? )
3. uxplay version.

GStreamer-1.24.0 broke something on UxPlay, and it was too noticed too late to fix in Ubuntu- 24.04LTS  release if that is what you are using,  but an update has been scheduled in ubuntu-24.04.     You can use the  github release uxplay-1.68.3
or current.

**Comment by gptlang** [comment]
> Distribution

Fedora 40

> gstreamer

`1.24.4-1`

> uxplay

`UxPlay version 1.68`


Worked after building from source.
